package.json

```diff
@@ -265,7 +265,7 @@
     "@metamask/browser-passworder": "^4.3.0",
     "@metamask/contract-metadata": "^2.5.0",
     "@metamask/controller-utils": "^11.4.0",
-    "@metamask/design-tokens": "^6.0.1",
+    "@metamask/design-tokens": "^7.0.0",
```

Comment:
Upgrading design tokens to v7 https://github.com/MetaMask/metamask-design-system/blob/main/packages/design-tokens/MIGRATION.md#from-version-600-to-700